### ISSUE TITLE
How to use ImageDataGenerator when I have multi labels? datagen.flow(X_train,Y_train),but I have 2 labels ,[[label1,label2]...] don't work! 

### ISSUE THREAD

**Issue by kesonyk** [fchollet/keras#2964]
Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by kesonyk** [comment]
(X_train,coarseLabel_train,fineLabel_train),(X_test,coarseLabel_test,fineLabel_test)=load_data_2label()
how to use datagen.flow( )with 2 label? Thx
@fchollet

**Comment by iamhankai** [comment]
any solution?

**Comment by sharkdeng** [comment]
I am facing the same issue. what is your solution?
